aws-sdk-resources/lib/aws-sdk-resources/services/s3/encryption/io_decrypter.rb

```diff
@@ -4,19 +4,20 @@ module Encryption
       # @api private
       class IODecrypter
 
-        # @param [OpenSSL::Cipher] cipher
-        # @param [#write] io An IO-like object that responds to {#write}.
-        def initialize(cipher, io)
-          @orig_cipher = cipher.clone
-          @cipher = cipher.clone
-          @io = io
+        # @option options [required, #write] :io
+        #   An IO-like object that responds to {#write}.
+        # @option options [required, OpenSSL::Cipher] :cipher
+        def initialize(options = {})
```

Comment:
By previous convention, we may want to consider releasing this feature as `2.4.0`. Arguably minor enough to not bother, but it's technically a breaking change of a private API.